tests/show_ip_int_test.py

```diff
@@ -0,0 +1,217 @@
+    'sudo sysctl -w net.ipv6.conf.default.addr_gen_mode=1',
+    # dummy physical port
+    'sudo ip link add Ethernet0 type dummy',
+    'sudo ip addr add 20.1.1.1/24 dev Ethernet0',
+    'sudo ip -6 address add dev Ethernet0 scope link fe80::64be:a1ff:fe85:c6c4/64',
+    'sudo ip -6 addr add aa00::1/64 dev Ethernet0',
+    'sudo ip link set Ethernet0 up',
+
+    #dummy portchannel
+    'sudo ip link add PortChannel0001 type dummy',
+    'sudo ip addr add 30.1.1.1/24 dev PortChannel0001',
+    'sudo ip -6 address add dev PortChannel0001 scope link fe80::cc8d:60ff:fe08:139f/64',
+    'sudo ip -6 addr add ab00::1/64 dev PortChannel0001',
+    'sudo ip link set PortChannel0001 up',
+
+    #dummy Vlan interface
+    'sudo ip link add Vlan100 type dummy',
+    'sudo ip addr add 40.1.1.1/24 dev Vlan100',
+    'sudo ip -6 address add dev Vlan100 scope link fe80::c029:3fff:fe41:cf56/64',
+    'sudo ip -6 addr add cc00::1/64 dev Vlan100',
+    'sudo ip link set Vlan100 up',
+
+]
+cleanup_ip_intfs_single_asic = [
+    'sudo ip link del Ethernet0',
+    'sudo ip link del PortChannel0001',
+    'sudo ip link del Vlan100',
+    'sudo sysctl -w net.ipv6.conf.default.addr_gen_mode=1',
+    'sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1'
+
+]
+
+show_ip_intf = """Interface        Master    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
+---------------  --------  -------------------  ------------  --------------  -------------
+Ethernet0                  20.1.1.1/24          error/down    T2-Peer         20.1.1.5
+PortChannel0001            30.1.1.1/24          error/down    T0-Peer         30.1.1.5
+Vlan100                    40.1.1.1/24          error/down    N/A             N/A
+lo                         127.0.0.1/8          error/down    N/A             N/A"""
+
+show_ipv6_intf = """Interface        Master    IPv4 address/mask                             Admin/Oper    BGP Neighbor    Neighbor IP
+---------------  --------  --------------------------------------------  ------------  --------------  -------------
+Ethernet0                  aa00::1/64                                    error/down    N/A             N/A
+                           fe80::64be:a1ff:fe85:c6c4%Ethernet0/64                      N/A             N/A
+PortChannel0001            ab00::1/64                                    error/down    N/A             N/A
+                           fe80::cc8d:60ff:fe08:139f%PortChannel0001/64                N/A             N/A
+Vlan100                    cc00::1/64                                    error/down    N/A             N/A
+                           fe80::c029:3fff:fe41:cf56%Vlan100/64                        N/A             N/A
+lo                         ::1/128                                       error/down    N/A             N/A"""
+
+show_ipv4_intf_with_multple_ips = """Interface        Master    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
```

Comment:
Suggest changing to the following for better readability/alignment with following lines:

```python
show_ipv4_intf_with_multple_ips = """\
Interface        Master    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
```

Reply:
Fixed in latest commit